save-api/src/main/kotlin/com/saveourtool/save/api/utils/RequestUtils.kt

```diff
@@ -101,60 +84,17 @@ suspend fun HttpClient.uploadAdditionalFile(
 }.body()
 
 /**
- * @return list of existing standard test suites
- */
-suspend fun HttpClient.getStandardTestSuites(
-): List<TestSuiteDto> = getRequestWithAuthAndJsonContentType(
-    "${Backend.url}/api/$v1/allStandardTestSuites"
-).body()
-
-/**
- * Submit execution, according [executionType] with list of [additionalFiles]
+ * Submit execution
  *
- * @param executionType type of requested execution git/standard
- * @param executionRequest execution request
- * @param additionalFiles list of additional files for execution
+ * @param runExecutionRequest execution request
  * @return HttpResponse
  */
-@OptIn(InternalAPI::class)
 @Suppress("TOO_LONG_FUNCTION")
-suspend fun HttpClient.submitExecution(executionType: TestingType, executionRequest: ExecutionRequestBase, additionalFiles: List<ShortFileInfo>?): HttpResponse {
-    val endpoint = if (executionType == TestingType.PRIVATE_TESTS) {
-        "/api/$v1/submitExecutionRequest"
-    } else {
-        "/api/$v1/executionRequestStandardTests"
-    }
-    return this.post {
-        url("${Backend.url}$endpoint")
-        header("X-Authorization-Source", UserInformation.source)
-        val formDataHeaders = Headers.build {
-            append(HttpHeaders.ContentType, ContentType.Application.Json)
-        }
-        setBody(MultiPartFormDataContent(formData {
-            if (executionType == TestingType.PRIVATE_TESTS) {
-                append(
-                    "executionRequest",
-                    json.encodeToString(executionRequest as ExecutionRequest),
-                    formDataHeaders
-                )
-            } else {
-                append(
-                    "execution",
-                    json.encodeToString(executionRequest as ExecutionRequestForStandardSuites),
-                    formDataHeaders
-                )
-            }
-            additionalFiles?.forEach { shortFileInfo ->
-                append(
-                    "file",
-                    json.encodeToString(shortFileInfo),
-                    Headers.build {
-                        append(HttpHeaders.ContentType, ContentType.Application.Json)
-                    }
-                )
-            }
-        }))
-    }
+suspend fun HttpClient.submitExecution(runExecutionRequest: RunExecutionRequest): HttpResponse = this.post {
+    url("${Backend.url}/api/$v1/run/trigger")
```

Comment:
I believe this should contain `testingType` as a request param?

Reply:
yes

Reply:
returned back